proposals/dictionary-expressions.md

```diff
@@ -54,71 +55,124 @@ Choices here would have implications regarding potential syntactic ambiguities,
+
+### Answered question 1
+
+Can a dictionary type value be created without using a key_value_pair_element?  For example are the following legal:
 
 ```c#
-Dictionary<string, int> nameToAge = ["mads": 21, .. existingDict.Where(kvp => kvp.Value >= 21)];
+Dictionary<string, int> d1 = [existingKvp];
+Dictionary<string, int> d2 = [.. otherDict];
 ```
 
+Note: the element `KeyValuePair<K1,V1>` types need not be identical to the `KeyValuePair<K2,V2>` type of the destination dictionary type.  They simply must be convertible to the `V1 this[K1 key] { ... }` indexer provided by the dictionary.
+
+Yes.  These are legal: [LDM-2024-03-11](https://github.com/dotnet/csharplang/blob/main/meetings/2024/LDM-2024-03-11.md#conclusions)
+
+### Answered question 2
+
+Can you spread a *non dictionary type* when producing a dictionary type'd value.  For example:
+
+```c#
+Dictionary<string, int> nameToAge = ["mads": 21, .. existingListOfKVPS];
+``` 
+
 **Resolution:** *Spread elements* of key-value pair collections will be supported in dictionary expressions. [LDM-2024-03-11](https://github.com/dotnet/csharplang/blob/main/meetings/2024/LDM-2024-03-11.md#conclusions)
 
-### Open question 3
+### Answered question 3
 
-How far do we want to take this KeyValuePair representation of things? Do we allow *dictionary elements* when producing normal collections? For example, should the following be allowed:
+How far do we want to take this KeyValuePair representation of things? Do we allow *key value pair elements* when producing normal collections? For example, should the following be allowed:
 
 ```c#
 List<KeyValuePair<string, int>> = ["mads": 21];
 ```
 
-Or should the presence of `k:v` element mandate some dictionary type as the receiver.  Note: the above sort of API is not uncommon.  Roslyn itself contains many apis that permissively *receive* an `IEnumerable<KeyValuePair<string, int>>`.
+**Resolution:** *Key value pair elements* will be supported in collection expressions for collection types that have a key-value pair element type. [LDM-2024-03-11](https://github.com/dotnet/csharplang/blob/main/meetings/2024/LDM-2024-03-11.md#conclusions)
 
-Importantly, we do not believe it wise to *require* the presence of a `k:v` element to produce a dictionary instance.  For example, we believe it very reasonable and desirable that someone be able to write:
+### Answered question 4
+
+Dictionaries provide two ways of initializing their contents.  A restrictive `.Add`-oriented form that throws when a key is already present in the dictionary, used by collection initializers, and a permissive indexer-oriented form which does not, used by dictionary initializers.  The restrictive form is useful for catching mistakes ("oops, I didn't intend to add the same thing twice!"), but is limiting *especially* in the spread case.  For example:
```

Comment:
The "used by ..." confused me. For `new() { ... }` initializers, dictionaries use `Add()` - see [sharplab.io](https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA+ABADAAgwRhQG4BYAKHIBEBLMAF2ogDsBDKATwB5qm6AaHDzoA+HABMcAXhxMYAdwAUAShwBvNTnwCATDgC+A9doEBmfYc0CU+/USA=).

Consider removing "used by ...":
```suggestion
Dictionaries provide two ways of initializing their contents.  A restrictive `Add`-oriented form that throws when a key is already present in the dictionary, and a permissive indexer-oriented form which does not.  The restrictive form is useful for catching mistakes ("oops, I didn't intend to add the same thing twice!"), but is limiting *especially* in the spread case.  For example:
```